### ISSUE TITLE
Cassandra Integration tests fail to start in intellij

### ISSUE THREAD

**Issue by gsheasby** [palantir/atlasdb#1754]
Observed on develop, and reproduced in 0.35.0 (it worked as of 0.34.0).

Running any Cassandra integration test class from intellij gives the error below. Running through gradle is fine (but iteration is a fair bit slower and you can't debug).

```
com.palantir.docker.compose.execution.DockerExecutionException: 'docker-compose ps -q The CASSANDRA_MAX_HEAP_SIZE variable is not set. Defaulting to a blank string.' returned exit code 1
The output was:
The CASSANDRA_MAX_HEAP_SIZE variable is not set. Defaulting to a blank string.
The CASSANDRA_HEAP_NEWSIZE variable is not set. Defaulting to a blank string.
No such service: The CASSANDRA_MAX_HEAP_SIZE variable is not set. Defaulting to a blank string.

	at com.palantir.docker.compose.execution.Command.lambda$throwingOnError$12(Command.java:60)
	at com.palantir.docker.compose.execution.Command$$Lambda$35/1686100174.handle(Unknown Source)
	at com.palantir.docker.compose.execution.Command.execute(Command.java:50)
	at com.palantir.docker.compose.execution.DefaultDockerCompose.id(DefaultDockerCompose.java:169)
	at com.palantir.docker.compose.execution.DelegatingDockerCompose.id(DelegatingDockerCompose.java:102)
	at com.palantir.docker.compose.execution.RetryingDockerCompose.id(RetryingDockerCompose.java:22)
	at com.palantir.docker.compose.connection.Container.state(Container.java:108)
	at com.palantir.docker.compose.connection.waiting.ClusterHealthCheck.lambda$nativeHealthChecks$21(ClusterHealthCheck.java:56)
	at com.palantir.docker.compose.connection.waiting.ClusterHealthCheck$$Lambda$55/1861781750.isClusterHealthy(Unknown Source)
	at com.palantir.docker.compose.connection.waiting.ClusterWait.lambda$weHaveSuccess$22(ClusterWait.java:57)
	at com.palantir.docker.compose.connection.waiting.ClusterWait$$Lambda$56/1043208434.call(Unknown Source)
	at com.jayway.awaitility.core.CallableCondition$ConditionEvaluationWrapper.call(CallableCondition.java:99)
	at com.jayway.awaitility.core.CallableCondition$ConditionEvaluationWrapper.call(CallableCondition.java:85)
	at com.jayway.awaitility.core.ConditionAwaiter$1.run(ConditionAwaiter.java:54)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by gmaretic** [comment]
I have those variables set in my environment as a workaround (CASSANDRA_MAX_HEAP_SIZE=512m and CASSANDRA_HEAP_NEWSIZE=64m). Maybe it would be enough to document the need for this?

**Comment by berler** [comment]
Couldn't you use default values if they are not set in the environment?

**Comment by rhero** [comment]
We should fix this so it just starts up and works.

**Comment by gsheasby** [comment]
Fixed by #1802.
